src/Explorer/Panes/PanelInfoErrorComponent.tsx

```diff
@@ -8,38 +8,47 @@ export interface PanelInfoErrorProps {
   link?: string;
   linkText?: string;
   openNotificationConsole?: () => void;
+  formError?: boolean;
 }
 
-export const PanelInfoErrorComponent: React.FunctionComponent<PanelInfoErrorProps> = (
-  props: PanelInfoErrorProps
-): JSX.Element => {
+export const PanelInfoErrorComponent: React.FunctionComponent<PanelInfoErrorProps> = ({
+  message,
+  messageType,
+  showErrorDetails,
+  link,
+  linkText,
+  openNotificationConsole,
+  formError = true,
```

Comment:
Move `formError` to the parent component:
`{formError && <PanelInfoErrorComponent ... />}`